### DISCUSSION TITLE
How do I build it?

### DISCUSSION THREAD

**Discussion by ItsTerm1n4l** [bk138/gromit-mpx#218] [Q&A] [answered]
Im on Manjaro and I wanted to change the cursor xpm file but the build fails at `make`
It installed fine from the aur but even if I manually download from the aur and make it  it fails.

**Comment by bk138** [comment]
Hi, please have a look at https://github.com/bk138/gromit-mpx?tab=readme-ov-file#building-it

**Reply by ItsTerm1n4l** [reply]
I followed that and the `cmake ..` command runs fine, but then when I `cd` into the `build` directory and run `make` I get errors.
I did try the most recent release and a git clone and neither worked.

**Comment by ItsTerm1n4l** [comment, marked as answer]
Fixed, my patch that I was applying to the build removed 2 lines from the cursor xpm and that broke it. mb.
sorry to waste your time.
